### ISSUE TITLE
Phantom 1.9.8 Crash (Mac OS X Yosemite)

### ISSUE THREAD

**Issue by blimmer** [ariya/phantomjs#12737]
I'm able to reproduce this every time I try to run my test suite (Karma + PhantomJS) on my OS X 10.10.0 machine. Running the same set of tests on Phantom 1.9.8 on Linux (Ubuntu 12.04 LTS) works without problems.

Here's the dump:

```
Operating system: Mac OS X
                  10.10.0 14A389
CPU: x86
     GenuineIntel family 6 model 70 stepping 1
     8 CPUs

Crash reason:  EXC_BAD_INSTRUCTION / EXC_I386_INVOP
Crash address: 0x8c26a

Thread 0 (crashed)
 0  phantomjs!__ZN3WTF10fastMallocEm + 0x1da
    eip = 0x0008c26a   esp = 0xbfffc200   ebp = 0xbfffc238   ebx = 0x0000003f
    esi = 0x0008c09e   edi = 0x00000040   eax = 0x00000000   ecx = 0x01844e3c
    edx = 0xf52c0f00   efl = 0x00210246
    Found by: given as instruction pointer in context
 1  phantomjs!__ZN7WebCore13TextCodecUTF86decodeEPKcmbbRb + 0x2f
    eip = 0x00592956   esp = 0xbfffc240   ebp = 0xbfffc298
    Found by: previous frame's frame pointer
 2  phantomjs!__ZN7WebCore19TextResourceDecoder6decodeEPKcm + 0x206
    eip = 0x004deb7b   esp = 0xbfffc2a0   ebp = 0xbfffc2f8
    Found by: previous frame's frame pointer
 3  phantomjs!__ZN7WebCore12CachedScript6scriptEv + 0x50
    eip = 0x004a9bbd   esp = 0xbfffc300   ebp = 0xbfffc338
    Found by: previous frame's frame pointer
 4  phantomjs!__ZNK7WebCore26CachedScriptSourceProvider4dataEv + 0x13
    eip = 0x003288c4   esp = 0xbfffc340   ebp = 0xbfffc348
    Found by: previous frame's frame pointer
 5  phantomjs!__ZN3JSC5Lexer7setCodeERKNS_10SourceCodeERNS_11ParserArenaE + 0x31
    eip = 0x0013b748   esp = 0xbfffc350   ebp = 0xbfffc368
    Found by: previous frame's frame pointer
 6  phantomjs!__ZN3JSC6Parser5parseEPNS_12JSGlobalDataEPNS_18FunctionParametersENS_18JSParserStrictnessENS_12JSParserModeEPiPNS_7UStringE + 0x88
    eip = 0x0013f80d   esp = 0xbfffc370   ebp = 0xbfffc3b8
    Found by: previous frame's frame pointer
 7  phantomjs!__ZN3JSC6Parser5parseINS_16FunctionBodyNodeEEEN3WTF10PassRefPtrIT_EEPNS_14JSGlobalObjectEPNS_8DebuggerEPNS_9ExecStateERKNS_10SourceCodeEPNS_18FunctionParametersENS_18JSParserStrictnessEPPNS_8JSObjectE + 0x5b
    eip = 0x0015e090   esp = 0xbfffc3c0   ebp = 0xbfffc418
    Found by: previous frame's frame pointer
 8  phantomjs!__ZN3JSC18FunctionExecutable22compileForCallInternalEPNS_9ExecStateEPNS_14ScopeChainNodeE + 0x87
    eip = 0x0015c1f6   esp = 0xbfffc420   ebp = 0xbfffc5f8
    Found by: previous frame's frame pointer
 9  phantomjs!_cti_vm_lazyLinkCall + 0x61
    eip = 0x00117f72   esp = 0xbfffc600   ebp = 0xbfffc648
    Found by: previous frame's frame pointer
10  0xa2a105f
    eip = 0x0a2a1060   esp = 0xbfffc650   ebp = 0xbfffc698
    Found by: previous frame's frame pointer
11  phantomjs!__ZN3JSC11Interpreter11executeCallEPNS_9ExecStateEPNS_8JSObjectENS_8CallTypeERKNS_8CallDataENS_7JSValueERKNS_7ArgListE + 0x533
    eip = 0x000d90c6   esp = 0xbfffc6a0   ebp = 0xbfffc728
    Found by: previous frame's frame pointer
12  phantomjs!__ZN3JSC4callEPNS_9ExecStateENS_7JSValueENS_8CallTypeERKNS_8CallDataES2_RKNS_7ArgListE + 0x48
    eip = 0x00150e79   esp = 0xbfffc730   ebp = 0xbfffc758
    Found by: previous frame's frame pointer
13  phantomjs!__ZN7WebCore15ScheduledAction24executeFunctionInContextEPN3JSC14JSGlobalObjectENS1_7JSValueEPNS_22ScriptExecutionContextE + 0x205
    eip = 0x00205752   esp = 0xbfffc760   ebp = 0xbfffc848
    Found by: previous frame's frame pointer
14  phantomjs!__ZN7WebCore15ScheduledAction7executeEPNS_8DocumentE + 0xc7
    eip = 0x00205a7e   esp = 0xbfffc850   ebp = 0xbfffc898
    Found by: previous frame's frame pointer
15  phantomjs!__ZN7WebCore15ScheduledAction7executeEPNS_22ScriptExecutionContextE + 0x32
    eip = 0x00205b65   esp = 0xbfffc8a0   ebp = 0xbfffc8b8
    Found by: previous frame's frame pointer
16  phantomjs!__ZN7WebCore8DOMTimer5firedEv + 0x172
    eip = 0x00500395   esp = 0xbfffc8c0   ebp = 0xbfffc8f8
    Found by: previous frame's frame pointer
17  phantomjs!__ZN7WebCore12ThreadTimers24sharedTimerFiredInternalEv + 0x8f
    eip = 0x005977b8   esp = 0xbfffc900   ebp = 0xbfffc938
    Found by: previous frame's frame pointer
18  phantomjs!__ZN7WebCore12ThreadTimers16sharedTimerFiredEv + 0x15
    eip = 0x00597722   esp = 0xbfffc940   ebp = 0xbfffc948
    Found by: previous frame's frame pointer
19  phantomjs!__ZN7QObject5eventEP6QEvent + 0x3b
    eip = 0x011eb640   esp = 0xbfffc950   ebp = 0xbfffc9f8
    Found by: previous frame's frame pointer
20  phantomjs!__ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent + 0x133
    eip = 0x00beb624   esp = 0xbfffca00   ebp = 0xbfffca18
    Found by: previous frame's frame pointer
21  phantomjs!__ZN12QApplication6notifyEP7QObjectP6QEvent + 0x294
    eip = 0x00beb8dd   esp = 0xbfffca20   ebp = 0xbfffcd78
    Found by: previous frame's frame pointer
22  phantomjs!__ZN16QCoreApplication14notifyInternalEP7QObjectP6QEvent + 0x5d
    eip = 0x011da05c   esp = 0xbfffcd80   ebp = 0xbfffcda8
    Found by: previous frame's frame pointer
23  phantomjs!__Z23qt_sendSpontaneousEventP7QObjectP6QEvent + 0x35
    eip = 0x00beb016   esp = 0xbfffcdb0   ebp = 0xbfffcdc8
    Found by: previous frame's frame pointer
24  phantomjs + 0xbaab18
    eip = 0x00babb19   esp = 0xbfffcdd0   ebp = 0xbfffcdf8
    Found by: previous frame's frame pointer

Thread 1
 0  libsystem_kernel.dylib + 0x1a8d2
    eip = 0x9b2058d2   esp = 0xb009267c   ebp = 0xb00926f8   ebx = 0x00000000
    esi = 0x9571d658   edi = 0x9573e0a0   eax = 0x00000171   ecx = 0xb009267c
    edx = 0x9b2058d2   efl = 0x00000246
    Found by: given as instruction pointer in context
 1  libdispatch.dylib + 0x43a1
    eip = 0x9571d3a2   esp = 0xb0092700   ebp = 0xb0092f08
    Found by: previous frame's frame pointer

Thread 2
 0  libsystem_kernel.dylib + 0x13b4e
    eip = 0x9b1feb4e   esp = 0xb039eecc   ebp = 0xb039ef18   ebx = 0x5714fa0f
    esi = 0x0000261a   edi = 0x00000000   eax = 0xffffffa6   ecx = 0xb039eecc
    edx = 0x9b1feb4e   efl = 0x00000202
    Found by: given as instruction pointer in context
 1  libsystem_c.dylib + 0x8580c
    eip = 0x9ab8680d   esp = 0xb039ef20   ebp = 0xb039ef48
    Found by: previous frame's frame pointer
 2  phantomjs!__ZN3WTF17TCMalloc_PageHeap15scavengerThreadEv + 0xc2
    eip = 0x0008a343   esp = 0xb039ef50   ebp = 0xb039ef78
    Found by: previous frame's frame pointer
 3  phantomjs!__ZN3WTF17TCMalloc_PageHeap18runScavengerThreadEPv + 0x10
    eip = 0x0008a379   esp = 0xb039ef80   ebp = 0xb039ef88
    Found by: previous frame's frame pointer
 4  libsystem_pthread.dylib + 0x2ece
    eip = 0x91cf3ecf   esp = 0xb039ef90   ebp = 0xb039efa8
    Found by: previous frame's frame pointer
 5  libsystem_pthread.dylib + 0x2e44
    eip = 0x91cf3e45   esp = 0xb039efb0   ebp = 0xb039efc8
    Found by: previous frame's frame pointer
 6  libsystem_pthread.dylib + 0xf0d
    eip = 0x91cf1f0e   esp = 0xb039efd0   ebp = 0xb039efec
    Found by: previous frame's frame pointer

Thread 3
 0  libsystem_kernel.dylib + 0x1986e
    eip = 0x9b20486e   esp = 0xb0420cec   ebp = 0xb0420d18   ebx = 0x0185a9a0
    esi = 0x088abaf4   edi = 0x00000012   eax = 0x00140197   ecx = 0xb0420cec
    edx = 0x9b20486e   efl = 0x00000246
    Found by: given as instruction pointer in context
 1  phantomjs!__Z14qt_safe_selectiP6fd_setS0_S0_PK7timeval + 0x37
    eip = 0x011fa568   esp = 0xb0420d20   ebp = 0xb0420d68
    Found by: previous frame's frame pointer
 2  phantomjs!__ZN20QEventDispatcherUNIX6selectEiP6fd_setS1_S1_P7timeval + 0x2c
    eip = 0x011fb701   esp = 0xb0420d70   ebp = 0xb0420d88
    Found by: previous frame's frame pointer
 3  phantomjs!__ZN27QEventDispatcherUNIXPrivate8doSelectE6QFlagsIN10QEventLoop17ProcessEventsFlagEEP7timeval + 0x471
    eip = 0x011fc618   esp = 0xb0420d90   ebp = 0xb0420e78
    Found by: previous frame's frame pointer
 4  phantomjs!__ZN20QEventDispatcherUNIX13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE + 0xe9
    eip = 0x011fca26   esp = 0xb0420e80   ebp = 0xb0420eb8
    Found by: previous frame's frame pointer
 5  phantomjs!__ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE + 0x210
    eip = 0x011d7ae3   esp = 0xb0420ec0   ebp = 0xb0420ef8
    Found by: previous frame's frame pointer
 6  phantomjs!__ZN7QThread4execEv + 0xca
    eip = 0x01102e27   esp = 0xb0420f00   ebp = 0xb0420f38
    Found by: previous frame's frame pointer
 7  phantomjs!__ZN14QThreadPrivate5startEPv + 0x1fc
    eip = 0x01105547   esp = 0xb0420f40   ebp = 0xb0420f88
    Found by: previous frame's frame pointer
 8  libsystem_pthread.dylib + 0x2ece
    eip = 0x91cf3ecf   esp = 0xb0420f90   ebp = 0xb0420fa8
    Found by: previous frame's frame pointer
 9  libsystem_pthread.dylib + 0x2e44
    eip = 0x91cf3e45   esp = 0xb0420fb0   ebp = 0xb0420fc8
    Found by: previous frame's frame pointer
10  libsystem_pthread.dylib + 0xf0d
    eip = 0x91cf1f0e   esp = 0xb0420fd0   ebp = 0xb0420fec
    Found by: previous frame's frame pointer

Thread 4
 0  libsystem_kernel.dylib + 0x1986e
    eip = 0x9b20486e   esp = 0xb04a2cec   ebp = 0xb04a2d18   ebx = 0x0185a9a0
    esi = 0x038414f4   edi = 0x00000022   eax = 0x00140197   ecx = 0xb04a2cec
    edx = 0x9b20486e   efl = 0x00000202
    Found by: given as instruction pointer in context
 1  phantomjs!__Z14qt_safe_selectiP6fd_setS0_S0_PK7timeval + 0x1b7
    eip = 0x011fa6e8   esp = 0xb04a2d20   ebp = 0xb04a2d68
    Found by: previous frame's frame pointer
 2  phantomjs!__ZN20QEventDispatcherUNIX6selectEiP6fd_setS1_S1_P7timeval + 0x2c
    eip = 0x011fb701   esp = 0xb04a2d70   ebp = 0xb04a2d88
    Found by: previous frame's frame pointer
 3  phantomjs!__ZN27QEventDispatcherUNIXPrivate8doSelectE6QFlagsIN10QEventLoop17ProcessEventsFlagEEP7timeval + 0x471
    eip = 0x011fc618   esp = 0xb04a2d90   ebp = 0xb04a2e78
    Found by: previous frame's frame pointer
 4  phantomjs!__ZN20QEventDispatcherUNIX13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE + 0xe9
    eip = 0x011fca26   esp = 0xb04a2e80   ebp = 0xb04a2eb8
    Found by: previous frame's frame pointer
 5  phantomjs!__ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE + 0x210
    eip = 0x011d7ae3   esp = 0xb04a2ec0   ebp = 0xb04a2ef8
    Found by: previous frame's frame pointer
 6  phantomjs!__ZN7QThread4execEv + 0xca
    eip = 0x01102e27   esp = 0xb04a2f00   ebp = 0xb04a2f38
    Found by: previous frame's frame pointer
 7  phantomjs!__ZN14QThreadPrivate5startEPv + 0x1fc
    eip = 0x01105547   esp = 0xb04a2f40   ebp = 0xb04a2f88
    Found by: previous frame's frame pointer
 8  libsystem_pthread.dylib + 0x2ece
    eip = 0x91cf3ecf   esp = 0xb04a2f90   ebp = 0xb04a2fa8
    Found by: previous frame's frame pointer
 9  libsystem_pthread.dylib + 0x2e44
    eip = 0x91cf3e45   esp = 0xb04a2fb0   ebp = 0xb04a2fc8
    Found by: previous frame's frame pointer
10  libsystem_pthread.dylib + 0xf0d
    eip = 0x91cf1f0e   esp = 0xb04a2fd0   ebp = 0xb04a2fec
    Found by: previous frame's frame pointer

Thread 5
 0  libsystem_kernel.dylib + 0x13a2e
    eip = 0x9b1fea2e   esp = 0xb05a5ffc   ebp = 0xb05a6038   ebx = 0xffffffff
    esi = 0x07000806   edi = 0x00008003   eax = 0xffffffe1   ecx = 0xb05a5ffc
    edx = 0x9b1fea2e   efl = 0x00000206
    Found by: given as instruction pointer in context
 1  CoreFoundation + 0x78ae5
    eip = 0x957c0ae6   esp = 0xb05a6040   ebp = 0xb05a6088
    Found by: previous frame's frame pointer
 2  CoreFoundation + 0x77ef8
    eip = 0x957bfef9   esp = 0xb05a6090   ebp = 0xb05a6d58
    Found by: previous frame's frame pointer
 3  CoreFoundation + 0x77695
    eip = 0x957bf696   esp = 0xb05a6d60   ebp = 0xb05a6e08
    Found by: previous frame's frame pointer
 4  CoreFoundation + 0x774fa
    eip = 0x957bf4fb   esp = 0xb05a6e10   ebp = 0xb05a6e38
    Found by: previous frame's frame pointer
 5  AppKit + 0x19b873
    eip = 0x97cde874   esp = 0xb05a6e40   ebp = 0xb05a6f88
    Found by: previous frame's frame pointer
 6  libsystem_pthread.dylib + 0x2ece
    eip = 0x91cf3ecf   esp = 0xb05a6f90   ebp = 0xb05a6fa8
    Found by: previous frame's frame pointer
 7  libsystem_pthread.dylib + 0x2e44
    eip = 0x91cf3e45   esp = 0xb05a6fb0   ebp = 0xb05a6fc8
    Found by: previous frame's frame pointer
 8  libsystem_pthread.dylib + 0xf0d
    eip = 0x91cf1f0e   esp = 0xb05a6fd0   ebp = 0xb05a6fec
    Found by: previous frame's frame pointer

Thread 6
 0  libsystem_kernel.dylib + 0x19852
    eip = 0x9b204852   esp = 0xb0620a9c   ebp = 0xb0628f88   ebx = 0x00000000
    esi = 0xa08263f8   edi = 0x9580f2c1   eax = 0x0014005d   ecx = 0xb0620a9c
    edx = 0x9b204852   efl = 0x00000282
    Found by: given as instruction pointer in context
 1  libsystem_pthread.dylib + 0x2ece
    eip = 0x91cf3ecf   esp = 0xb0628f90   ebp = 0xb0628fa8
    Found by: previous frame's frame pointer
 2  libsystem_pthread.dylib + 0x2e44
    eip = 0x91cf3e45   esp = 0xb0628fb0   ebp = 0xb0628fc8
    Found by: previous frame's frame pointer
 3  libsystem_pthread.dylib + 0xf0d
    eip = 0x91cf1f0e   esp = 0xb0628fd0   ebp = 0xb0628fec
    Found by: previous frame's frame pointer

Thread 7
 0  libsystem_kernel.dylib + 0x19516
    eip = 0x9b204516   esp = 0xb06aad7c   ebp = 0xb06aae08   ebx = 0x0415a3ec
    esi = 0x00000901   edi = 0x00000a00   eax = 0x00000131   ecx = 0xb06aad7c
    edx = 0x9b204516   efl = 0x00000246
    Found by: given as instruction pointer in context
 1  libsystem_pthread.dylib + 0x382a
    eip = 0x91cf482b   esp = 0xb06aae10   ebp = 0xb06aae28
    Found by: previous frame's frame pointer
 2  phantomjs!__ZN21QWaitConditionPrivate4waitEm + 0xae
    eip = 0x011060df   esp = 0xb06aae30   ebp = 0xb06aae68
    Found by: previous frame's frame pointer
 3  phantomjs!__ZN14QWaitCondition4waitEP6QMutexm + 0x76
    eip = 0x01105db7   esp = 0xb06aae70   ebp = 0xb06aae88
    Found by: previous frame's frame pointer
 4  phantomjs!__ZN3WTF15ThreadCondition9timedWaitERNS_5MutexEd + 0x50
    eip = 0x000902bf   esp = 0xb06aae90   ebp = 0xb06aaeb8
    Found by: previous frame's frame pointer
 5  phantomjs!__ZN3WTF12MessageQueueIN7WebCore16LocalStorageTaskEE14waitForMessageEv + 0x86
    eip = 0x0074a7f1   esp = 0xb06aaec0   ebp = 0xb06aaee8
    Found by: previous frame's frame pointer
 6  phantomjs!__ZN7WebCore18LocalStorageThread24threadEntryPointCallbackEPv + 0x43
    eip = 0x0074a534   esp = 0xb06aaef0   ebp = 0xb06aaf08
    Found by: previous frame's frame pointer
 7  phantomjs!__ZN3WTFL16threadEntryPointEPv + 0x4b
    eip = 0x00091e8c   esp = 0xb06aaf10   ebp = 0xb06aaf28
    Found by: previous frame's frame pointer
 8  phantomjs!__ZN3WTF13ThreadPrivate3runEv + 0x12
    eip = 0x00090127   esp = 0xb06aaf30   ebp = 0xb06aaf38
    Found by: previous frame's frame pointer
 9  phantomjs!__ZN14QThreadPrivate5startEPv + 0x1fc
    eip = 0x01105547   esp = 0xb06aaf40   ebp = 0xb06aaf88
    Found by: previous frame's frame pointer
10  libsystem_pthread.dylib + 0x2ece
    eip = 0x91cf3ecf   esp = 0xb06aaf90   ebp = 0xb06aafa8
    Found by: previous frame's frame pointer
11  libsystem_pthread.dylib + 0x2e44
    eip = 0x91cf3e45   esp = 0xb06aafb0   ebp = 0xb06aafc8
    Found by: previous frame's frame pointer
12  libsystem_pthread.dylib + 0xf0d
    eip = 0x91cf1f0e   esp = 0xb06aafd0   ebp = 0xb06aafec
    Found by: previous frame's frame pointer

Loaded modules:
0x00001000 - 0x01743fff  phantomjs  ???  (main)
0x025d3000 - 0x025eefff  csparser  ???
0x07a00000 - 0x07ab8fff  ColorSyncDeprecated.dylib  ???
0x0a1ce000 - 0x0a210fff  CloudDocs  ???
0x0cffc000 - 0x0d032fff  libssl.0.9.8.dylib  ???
0x90008000 - 0x90284fff  Security  ???
0x90285000 - 0x90287fff  libsystem_coreservices.dylib  ???
0x90288000 - 0x90288fff  libkeymgr.dylib  ???
0x90289000 - 0x90329fff  QD  ???
0x9032a000 - 0x9032cfff  libRadiance.dylib  ???
0x9032d000 - 0x9045ffff  UIFoundation  ???
0x90460000 - 0x9047bfff  OpenScripting  ???
0x9051e000 - 0x9055bfff  libsystem_network.dylib  ???
0x9055c000 - 0x9056ffff  libcmph.dylib  ???
0x90570000 - 0x90570fff  vecLib  ???
0x90571000 - 0x90573fff  loginsupport  ???
0x906eb000 - 0x90801fff  CoreText  ???
0x90802000 - 0x90858fff  libc++.1.dylib  ???
0x90859000 - 0x90865fff  libcommonCrypto.dylib  ???
0x90866000 - 0x908dcfff  SearchKit  ???
0x908dd000 - 0x908defff  libsystem_blocks.dylib  ???
0x908df000 - 0x908fcfff  Ubiquity  ???
0x908fd000 - 0x90af3fff  libicucore.A.dylib  ???
0x90afc000 - 0x90b05fff  CommonAuth  ???
0x90b06000 - 0x90b39fff  libsystem_m.dylib  ???
0x90b3a000 - 0x90b3efff  libGIF.dylib  ???
0x90b7a000 - 0x90b80fff  libsystem_trace.dylib  ???
0x90b81000 - 0x90b93fff  Sharing  ???
0x90e82000 - 0x90eddfff  LanguageModeling  ???
0x90eeb000 - 0x9102ffff  ImageIO  ???
0x91111000 - 0x91119fff  NetFS  ???
0x9111a000 - 0x91183fff  libcorecrypto.dylib  ???
0x91184000 - 0x9126dfff  libcrypto.0.9.8.dylib  ???
0x9126e000 - 0x91308fff  ColorSync  ???
0x91309000 - 0x91309fff  liblaunch.dylib  ???
0x91314000 - 0x9138afff  SecurityFoundation  ???
0x9138b000 - 0x91408fff  IOKit  ???
0x91409000 - 0x9140cfff  ServiceManagement  ???
0x9140d000 - 0x914f9fff  libvMisc.dylib  ???
0x914fa000 - 0x91610fff  DesktopServicesPriv  ???
0x91611000 - 0x91622fff  libbsm.0.dylib  ???
0x91623000 - 0x91646fff  libJPEG.dylib  ???
0x91647000 - 0x916c6fff  SystemConfiguration  ???
0x916d3000 - 0x916dbfff  libsystem_dnssd.dylib  ???
0x916e5000 - 0x916fafff  MultitouchSupport  ???
0x916fb000 - 0x91886fff  AudioToolbox  ???
0x91887000 - 0x9189bfff  ImageCapture  ???
0x91be3000 - 0x91ce7fff  libJP2.dylib  ???
0x91ce8000 - 0x91cecfff  CommonPanels  ???
0x91ced000 - 0x91cf0fff  Help  ???
0x91cf1000 - 0x91cf9fff  libsystem_pthread.dylib  ???
0x91e11000 - 0x91e20fff  OpenGL  ???
0x91e21000 - 0x91e23fff  libquarantine.dylib  ???
0x91e24000 - 0x91e90fff  DataDetectorsCore  ???
0x91e91000 - 0x91f18fff  OSServices  ???
0x91f19000 - 0x91f62fff  libFontRegistry.dylib  ???
0x9206a000 - 0x92160fff  libxml2.2.dylib  ???
0x92161000 - 0x9228efff  CoreUI  ???
0x9228f000 - 0x92321fff  CoreSymbolication  ???
0x92322000 - 0x92348fff  IconServices  ???
0x92349000 - 0x92351fff  FSEvents  ???
0x92352000 - 0x92360fff  libxar.1.dylib  ???
0x92361000 - 0x9238afff  libsystem_info.dylib  ???
0x9238b000 - 0x923aafff  libresolv.9.dylib  ???
0x923ab000 - 0x923defff  CoreServicesInternal  ???
0x92433000 - 0x924d0fff  Ink  ???
0x924da000 - 0x924eafff  LangAnalysis  ???
0x927bb000 - 0x927befff  libpam.2.dylib  ???
0x92a78000 - 0x92a9efff  libc++abi.dylib  ???
0x92a9f000 - 0x92ab2fff  CoreBluetooth  ???
0x92ab3000 - 0x92accfff  Kerberos  ???
0x92b76000 - 0x92b87fff  libsystem_coretls.dylib  ???
0x92b88000 - 0x92b9ffff  libsystem_asl.dylib  ???
0x92ba0000 - 0x92ba0fff  Accelerate  ???
0x92ba1000 - 0x92be8fff  AppleJPEG  ???
0x92be9000 - 0x92bedfff  libheimdal-asn1.dylib  ???
0x92d0b000 - 0x92d24fff  libsystem_malloc.dylib  ???
0x92d25000 - 0x92d4bfff  libPng.dylib  ???
0x92d4c000 - 0x92fdffff  CoreData  ???
0x92fe0000 - 0x93009fff  libRIP.A.dylib  ???
0x93099000 - 0x930f6fff  PrintCore  ???
0x930f7000 - 0x930f9fff  libsystem_configuration.dylib  ???
0x930fa000 - 0x93206fff  libvDSP.dylib  ???
0x933b8000 - 0x9342cfff  Heimdal  ???
0x9342d000 - 0x93459fff  ChunkingLibrary  ???
0x93495000 - 0x93499fff  libCoreVMClient.dylib  ???
0x9349a000 - 0x9349bfff  libDiagnosticMessagesClient.dylib  ???
0x9349c000 - 0x934f1fff  CoreAudio  ???
0x934f2000 - 0x93532fff  libauto.dylib  ???
0x93533000 - 0x9353cfff  libGFXShared.dylib  ???
0x938c4000 - 0x93919fff  HTMLRendering  ???
0x9391a000 - 0x939bffff  Metadata  ???
0x93c46000 - 0x93c49fff  libextension.dylib  ???
0x93c4a000 - 0x93c9dfff  libstdc++.6.dylib  ???
0x93c9e000 - 0x93cccfff  libarchive.2.dylib  ???
0x93ccd000 - 0x93ceafff  libCRFSuite.dylib  ???
0x93ceb000 - 0x93cebfff  Carbon  ???
0x93cec000 - 0x93d06fff  liblzma.5.dylib  ???
0x93d07000 - 0x93d39fff  GSS  ???
0x93d3a000 - 0x93eabfff  libBLAS.dylib  ???
0x93eac000 - 0x93eb7fff  CrashReporterSupport  ???
0x93eb8000 - 0x93ebdfff  libmacho.dylib  ???
0x93ebe000 - 0x93efafff  RemoteViewServices  ???
0x93efb000 - 0x93efdfff  SecurityHI  ???
0x9445a000 - 0x9445ffff  libcompiler_rt.dylib  ???
0x94527000 - 0x94532fff  NetAuth  ???
0x94533000 - 0x94573fff  Symbolication  ???
0x94574000 - 0x945d8fff  AE  ???
0x945da000 - 0x947b6fff  libobjc.A.dylib  ???
0x947b7000 - 0x9480afff  HIServices  ???
0x94b18000 - 0x94b3efff  libxpc.dylib  ???
0x94b3f000 - 0x94b4dfff  libz.1.dylib  ???
0x94b4e000 - 0x94b4efff  libOpenScriptingUtil.dylib  ???
0x954dc000 - 0x954e9fff  libbz2.1.0.dylib  ???
0x954ea000 - 0x954eafff  CoreServices  ???
0x954eb000 - 0x954f4fff  libsystem_notify.dylib  ???
0x954f5000 - 0x954f6fff  libremovefile.dylib  ???
0x954f7000 - 0x954f8fff  libSystem.B.dylib  ???
0x954f9000 - 0x95585fff  PerformanceAnalysis  ???
0x95586000 - 0x95686fff  LaunchServices  ???
0x95687000 - 0x956b6fff  DictionaryServices  ???
0x956b7000 - 0x956bdfff  libsystem_platform.dylib  ???
0x956d2000 - 0x95715fff  libGLU.dylib  ???
0x95716000 - 0x95718fff  libCVMSPluginSupport.dylib  ???
0x95719000 - 0x95740fff  libdispatch.dylib  ???
0x95747000 - 0x95747fff  ApplicationServices  ???
0x95748000 - 0x95afbfff  CoreFoundation  ???
0x95afc000 - 0x95e54fff  libmecabra.dylib  ???
0x95e55000 - 0x961b4fff  Foundation  ???
0x961b5000 - 0x965e8fff  FaceCore  ???
0x965e9000 - 0x965ecfff  libdyld.dylib  ???
0x96647000 - 0x96647fff  AudioUnit  ???
0x96648000 - 0x9665ffff  libLinearAlgebra.dylib  ???
0x96660000 - 0x9666dfff  OpenDirectory  ???
0x9671c000 - 0x96813fff  libFontParser.dylib  ???
0x96814000 - 0x96b12fff  CarbonCore  ???
0x96b13000 - 0x96b15fff  libCGXType.A.dylib  ???
0x96b16000 - 0x96b1ffff  DiskArbitration  ???
0x96b20000 - 0x96b24fff  libcache.dylib  ???
0x96b25000 - 0x96b26fff  liblangid.dylib  ???
0x96b27000 - 0x96cebfff  QuartzCore  ???
0x96cec000 - 0x96cecfff  libunc.dylib  ???
0x96ced000 - 0x97128fff  vImage  ???
0x97129000 - 0x97148fff  GenerationalStorage  ???
0x97af8000 - 0x97b27fff  CoreVideo  ???
0x97b38000 - 0x97b3cfff  IOSurface  ???
0x97b40000 - 0x97b42fff  libsystem_sandbox.dylib  ???
0x97b43000 - 0x9872ffff  AppKit  ???
0x98730000 - 0x987dffff  IOBluetooth  ???
0x98815000 - 0x98821fff  libkxld.dylib  ???
0x98822000 - 0x9887dfff  libTIFF.dylib  ???
0x9887e000 - 0x9888efff  libGL.dylib  ???
0x9888f000 - 0x988e0fff  OpenCL  ???
0x988e1000 - 0x9890bfff  libxslt.1.dylib  ???
0x9891a000 - 0x9891ffff  Print  ???
0x98a02000 - 0x98a25fff  Apple80211  ???
0x98a26000 - 0x98a2cfff  libsystem_networkextension.dylib  ???
0x98a2d000 - 0x98b1efff  libiconv.2.dylib  ???
0x98b6f000 - 0x98bdafff  CoreWiFi  ???
0x98bdb000 - 0x98fc3fff  libLAPACK.dylib  ???
0x99f66000 - 0x99fe0fff  ATS  ???
0x9a0b9000 - 0x9a0c3fff  CarbonSound  ???
0x9a0c4000 - 0x9a0cbfff  SpeechRecognition  ???
0x9a0f6000 - 0x9a3e8fff  CoreImage  ???
0x9a3e9000 - 0x9a5edfff  CFNetwork  ???
0x9a5ee000 - 0x9a9e4fff  CoreGraphics  ???
0x9a9e5000 - 0x9aa58fff  CoreWLAN  ???
0x9aa59000 - 0x9aaaafff  libcups.2.dylib  ???
0x9aaab000 - 0x9aab8fff  SpeechSynthesis  ???
0x9aab9000 - 0x9aaf8fff  NavigationServices  ???
0x9aaf9000 - 0x9ab00fff  libunwind.dylib  ???
0x9ab01000 - 0x9ab96fff  libsystem_c.dylib  ???
0x9ab97000 - 0x9af6dfff  HIToolbox  ???
0x9af71000 - 0x9b100fff  libsqlite3.dylib  ???
0x9b104000 - 0x9b1cdfff  Backup  ???
0x9b1ce000 - 0x9b1dcfff  SpeechRecognitionCore  ???
0x9b1dd000 - 0x9b1eafff  ProtocolBuffer  ???
0x9b1eb000 - 0x9b20afff  libsystem_kernel.dylib  ???
0x9b20b000 - 0x9b24bfff  libGLImage.dylib  ???
0x9b2b8000 - 0x9b2bcfff  TCC  ???
0x9b427000 - 0x9b42ffff  libCGCMS.A.dylib  ???
0x9b430000 - 0x9b46afff  DebugSymbols  ???
0x9b46b000 - 0x9b474fff  libcopyfile.dylib  ???
0x9bd83000 - 0x9bd9efff  CFOpenDirectory  ???
0x9bd9f000 - 0x9bda0fff  TrustEvaluationAgent  ???
0x9bda1000 - 0x9bda3fff  libsystem_secinit.dylib  ???
```

**Comment by scien** [comment]
+1

**Comment by bmustata** [comment]
+1

**Comment by Vitallium** [comment]
Please upgrade to PhantomJS 2.0. 1.9.x is no longer supported, and known to have many crasher bugs that are fixed in 2.0.
